app/assets/stylesheets/budget.css

```diff
@@ -0,0 +1,321 @@
+:root {
+  --angels_wings: #A2B38E;
+  --default_border: 2px solid;
+  --default_border_radius: 6px;
+  --default_font_size: 20px;
+  --desert_heat: #E7654C;
+  --desert_rock: #AAA092;
+  --gear_shift: #755B64;
+  --golden_desert: #EEA649;
+  --light_and_fluffy: #DAD3CB;
+  --now_what: #EAD39F;
+  --sacred_beam: #0C0730;
+  --tears_waiting: #C48772;
+  --white_whale: #F4E7C7;
+}
+
+* {
+  color: var(--sacred_beam);
+  font-family: arial, sans-serif;
+}
+
+body {
+  background-color: var(--white_whale);
+  font-size: var(--default_font_size);
+}
+
+p {
+  margin-left: 120px;
+  text-align: left;
+}
+
+table {
+  background-color: var(--now_what);
+  border-radius: var(--default_border_radius);
+  margin-left: 130px;
+  width: 80%;
+}
+
+table caption {
+  padding: 20px;
+  text-align: left;
+}
+
+table, th, td {
+  border: var(--default_border) var(--gear_shift);
+}
+
+td {
+  text-align: center;
+  vertical-align: center;
+}
+
+td a:link {
+  background-color: var(--sacred_beam);
+  border: var(--default_border) var(--sacred_beam);
+  border-radius: var(--default_border_radius);
+  color: var(--white_whale);
+  display: inline-block;
+  padding: 5px 14px;
+  text-align: center;
+  text-decoration: none;
+}
+
+td a:visited {
+  background-color: var(--sacred_beam);
+  border: var(--default_border) var(--sacred_beam);
+  color: var(--white_whale);
+}
+
+th {
+  height: 40px;
+}
+
+.box {
+  background-color: var(--now_what);
+  border: var(--default_border) var(--sacred_beam);
+  border-radius: var(--default_border_radius);
+  padding-bottom: 30px;
+  text-align: center;
+  margin-left: 360px;
+  margin-top: 100px;
+  width: 40%;
+}
+
+.button a:link{
+  background-color: var(--sacred_beam);
+  border: var(--default_border) var(--sacred_beam);
+  border-radius: 4px;
+  color: var(--white_whale);
+  display: inline-block;
+  font-size: var(--default_font_size);
+  margin-left: 80px;
+  padding: 2px 7px 2px 7px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.button a:visited {
+  background-color: var(--sacred_beam);
+  border: var(--default_border) var(--sacred_beam);
+  color: var(--white_whale);
+}
+
+.dropdown {
+  float: left;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  background-color: inherit;
+  border: none;
+  color: var(--white_whale);
+  font-family: inherit;
+  font-size: var(--default_font_size);
+  margin: 0;
+  outline: none;
+  padding: 14px 16px;
+}
+
+.navbar a:hover, .dropdown:hover .dropbtn {
+  background-color: var(--now_what);
+  color: var(--sacred_beam)
+}
+
+.dropdown-content {
+  background-color: var(--sacred_beam);
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  display: none;
+  position: absolute;
+  width: 120px;
+  z-index: 1;
+}
+
+div .dropdown-content a {
```

Comment:
Why is the first `div` needed here?

Reply:
Good question, whole dropdown menu restyled in Done in ad-css-styling sha: b2d6f90